server/src/controllers/metadata.controller.ts

```diff
@@ -52,7 +52,11 @@ export class MetadataController {
     const onboarded = data['onboarded'];
 
     if (process.env.NODE_ENV == 'production') {
-      if (process.env.CHECK_FOR_UPDATES) {
+      if (
```

Comment:
Could you update this as per the docs?
checking `process.env.CHECK_FOR_UPDATES === '1'` should do.

Reply:
I see, the default behaviour is to be enabled as per docs.